src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/client/manageusers/ManageUsersClient.kt

```diff
@@ -0,0 +1,42 @@
+package uk.gov.justice.digital.hmpps.hmppsbookavideolinkapi.client.manageusers
+
+import org.springframework.stereotype.Component
+import org.springframework.web.reactive.function.client.WebClient
+import org.springframework.web.reactive.function.client.WebClientResponseException
+import reactor.core.publisher.Mono
+
+@Component
+class ManageUsersClient(private val manageUsersApiWebClient: WebClient) {
+
+  fun getUsersDetails(username: String): UserDetailsDto? =
+    manageUsersApiWebClient
+      .get()
+      .uri("/users/{username}", username)
+      .retrieve()
+      .bodyToMono(UserDetailsDto::class.java)
+      .onErrorResume(WebClientResponseException.NotFound::class.java) { Mono.empty() }
+      .block()
+
+  fun getUsersEmail(username: String): EmailAddressDto? =
+    manageUsersApiWebClient
+      .get()
+      .uri("/users/{username}/email", username)
+      .retrieve()
+      .bodyToMono(EmailAddressDto::class.java)
+      .onErrorResume(WebClientResponseException.NotFound::class.java) { Mono.empty() }
+      .block()
+}
+
+data class UserDetailsDto(
```

Comment:
So I did intend to generate these but Open API/Swagger code generator didn't like the JSON provided by manage users API and I didn't think it was worth spending loads of time on it for two simple DTO's.